Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/DataStream/Cancellation/CancelCompleted.cs

```diff
@@ -0,0 +1,24 @@
+using Unity.Entities;
+
+namespace Anvil.Unity.DOTS.Entities.TaskDriver
+{
+    /// <summary>
+    /// Represents the completion of a Cancel Request from a generic perspective where only the <see cref="Entity"/>
+    /// is returned that completed it's cancellation flow.
+    /// </summary>
+    public readonly struct CancelCompleted : IEntityProxyInstance
```

Comment:
Better matches the tense in the rest of the naming in tasks

```suggestion
    public readonly struct CancelComplete : IEntityProxyInstance
```

Reply:
Fixed